package.json

```diff
@@ -24,7 +24,7 @@
       "git add"
     ]
   },
-  "author": "",
+  "author": "Magda",
```

Comment:
Zwróć proszę uwagę, że to jest projekt grupowy i nie ma jednego autora, dlatego, zostaw to pole puste, a jeżeli chcesz dodać siebie jako współautorkę to lepiej skorzystać z pola `contributors` https://flaviocopes.com/package-json/#contributors, ale należało by przekazać pozostałym członkom zespołu informację o dopisaniu się do tego pola